openmmml/models/torchmdnetpotential.py

```diff
@@ -0,0 +1,215 @@
+                                lengthScale=0.1 # angstrom to nm, 
+                                energyScale=4.184 # kcal/mol to kJ/mol)
+
+    During system creation you can enable CUDA graphs for a speed-up for small molecules:
+
+    >>>  system = potential.createSystem(pdb.topology, cudaGraphs=True)
+    The default is to enable this for TensorNet models.
+
+
+    You can also specify the molecule's total charge:
+
+    >>>  system = potential.createSystem(pdb.topology, charge=0)
+
+    Pretained AceFF models can be created directly:
+
+    >>> potential = MLPotential('aceff-1.0')
+    >>> potential = MLPotential('aceff-1.1')
+    """
+
+    def __init__(self, 
+                 name: str,
+                 modelPath: str,
+                 lengthScale: float,
+                 energyScale: float
+    ) -> None:
+        """
+        Initialize the TorchMDNetPotentialImpl.
+
+        Parameters
+        ----------
+        name : str
+            The name of the model.
+            'torchmdnet' for a local model file, or pretrained models are available: 'aceff-1.0' or 'aceff-1.1'. 
+        modelPath : str, optional
+            The path to the locally trained torchmdnet model if ``name`` is 'torchmdnet'.
+        lengthScale : float
+            The length conversion factor from the model units to nanometers. 
+            If not specified the default is 0.1 which corresponds to a model in angstrom
+        energyScale : float
+            The energy conversion factor from the model units to kJ/mol.
+            If not specified the default is 96.4916 which corresponds to a model in eV.
+   
+        """
+        self.name = name
+        self.modelPath = modelPath
+        self.lengthScale = lengthScale
+        self.energyScale = energyScale
+
+    def addForces(self,
+                  topology: openmm.app.Topology,
+                  system: openmm.System,
+                  atoms: Optional[Iterable[int]],
+                  forceGroup: int,
+                  **args):
+        # Load the TorchMDNet model.
+
+
+        try:
+            import torchmdnet
+            from torchmdnet.models.model import load_model
+        except ImportError as e:
+            raise ImportError(f"Failed to import torchmdnet please install from https://torchmd-net.readthedocs.io/en/latest/installation.html")
+        import torch
+        import openmmtorch
+
+        includedAtoms = list(topology.atoms())
+        if atoms is not None:
+            includedAtoms = [includedAtoms[i] for i in atoms]
+        numbers = torch.tensor([atom.element.atomic_number for atom in includedAtoms])
+        charge = torch.tensor([args.get('charge', 0)], dtype=torch.float32)
+
+
+        if self.name == 'torchmdnet':
+            # a local path to a torchmdnet checkpoint must be provided 
+            model_file_path = self.modelPath
+        else:
+            from huggingface_hub import hf_hub_download
```

Comment:
If it's not installed, catch the ImportError like above and tell them where to get it.